extensions/amp-image-slider/0.1/test/integration/test-amp-image-slider.js

```diff
@@ -176,7 +186,7 @@ config.run('amp-image-slider', function() {
           .then(() => {
             const sliders = doc.getElementsByTagName('amp-image-slider');
             expect(sliders.length).to.be.greaterThan(0);
-            slider = sliders[0];
+            slider = doc.querySelector('#slider');
             container_ = slider
                 .querySelector('.i-amphtml-image-slider-container');
             expect(!!container_).to.be.true;
```

Comment:
Commenting out previous code:

`expect(container_).to.not.be.null;`

You want to choose the most semantically correct assertion, you'll get better error messages that way when they fail as well.